src/interface/trsm.hpp

```diff
@@ -0,0 +1,432 @@
+
```

Comment:
Can you call this fine trsm_interface.hpp. Since the operation has trsm.hpp, when giving relative path it may get confusion

Reply:
Renamed to `interface/trsm_interface.hpp`